.github/workflows/smoke-tests.yml

```diff
@@ -5,12 +5,15 @@ defaults:
     shell: bash
 
 on:
+  workflow_dispatch:
   schedule:
-    - cron: '*/60 * * * *' # every 60 minutes
+    - cron: '45 * * * *' # Run prod on 45th minute
```

Comment:
45 minutes seems like a really long time for prod. Should we reduce it to, maybe, 5 minutes?

Reply:
Good thinking. We used to run it every 5 minutes, but we were getting a lot of Slacks about intermittent failures. Maybe now with the retries we can bump it back up